File: Sample.cpp
class Sample {
  void main() {
    int a = 0;
    if (a == 0) {
      printf("0");
    } else {
      printf("1");
    }
    try {
      throw "Exception"
      a++;
    }
    catch (const char *str) {
      a--;
    }
  }
}

File: Sample.py
class Sample:
	def main():
		a = 0
		if a == 0:
			print (0)
		else:
			print (1)
		try:
			raise Exception('')
			a = a + 1
		except:
			a = a - 1
		else:
			a = a * 2
